data/games/guilty_gear_xrd_rev_2/images/answer/index.js

```diff
@@ -1,146 +1,158 @@
 export default [
   {
-    imageId: '5P',
-    image: require('./5P.png'),
+    "imageId": "summoner_the_basilisk",
+    "image": require('./summoner_the_basilisk.png')
```

Comment:
I'm not the biggest fan of how all these diffs are present simply due to items being reordered, but I'm not going to NACK the pull-request.

Reply:
I'm not wild about it either but I couldn't come up with a good way to sort the images in a way that would map to normals -> specials -> supers, etc...

Reply:
The image indices already existed. We could just add the 2-4 images by hand?

Reply:
Eh, it's fine though.

I will look into a more deterministic way to generate image indices in the future.

Reply:
I can hand-edit these indices but I'd like to move away from hand-crafting files (especially the image indices). It's going to quickly become unwieldy as we add characters, doubly so for games one of us isn't familiar with.

Reply:
yeah, that's cool